test/__init__.py

```diff
@@ -49,3 +51,22 @@ def run_role_meta_main(self, meta_main_text):
         results = self._call_runner(role_path)
         shutil.rmtree(role_path)
         return results
+
+
+def run_ansible_lint(cwd, role_path=None, bin=None, env=None):
```

Comment:
@ssbarnea anything stopping us from making a fixture out of it?

Reply:
Probably not